deepchecks/vision/checks/data_integrity/abstract_property_outliers.py

```diff
@@ -45,7 +45,10 @@ class AbstractPropertyOutliers(SingleDatasetCheck):
     properties : List[Dict[str, Any]], default: None
         List of properties. Replaces the default deepchecks properties.
         Each property is dictionary with keys 'name' (str), 'method' (Callable) and 'output_type' (str),
-        representing attributes of said method. 'output_type' must be one of 'continuous'/'discrete'
+        representing attributes of said method. 'output_type' must be one of:
+        - 'numeric' - for continuous ordinal outputs.
+        - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
+          but that these numbers have not inherent value.
```

Comment:
```suggestion
          but that these numbers do not have inherent value.
```
Fix in other places as well :)

Reply:
Also, let's add a reference here to https://docs.deepchecks.com/stable/user-guide/vision/vision_properties.html.
You can use the :ref: keyword for that. 
Something like "for more on image / label properties, see the property guide" and have the words "property guide" be a url to the docs